release/7-3/ubi8/meta.json

```
@@ -3,10 +3,10 @@
     "UseLinuxVersion": false,
     "PackageFormat": "powershell${channelTag}-${PS_VERSION}-1.rh.x86_64.rpm",
     "SkipGssNtlmSspTests": true,
-    "osVersion": "Universal Base Image 8.4",
+    "osVersion": "Universal Base Image 8",
     "ShortDistroName": "ubi",
     "shortTags": [
-        {"Tag": "8.4"}
+        {"Tag": "8"}
```

Should this be 8.6 to match base image ubi8.6 used on Line 4 of the the 7-3 ubi 8 dockerfile
```suggestion
        {"Tag": "8.6"}
```